app/src/controllers/testItem/selectors.js

```diff
@@ -100,6 +100,8 @@ export const isTestItemsListSelector = createSelector(
 
 export const searchedTestItemsSelector = (state) => domainSelector(state).searchedItems;
 
+export const isSearchedItemsSelector = (state) => !isEmptyObject(searchedTestItemsSelector(state));
```

Comment:
This sounds more like `whether this selector is a searched items selector?`, so please rename to `isSearchWidgetItemsExistSelector` or smth like this.